tests/unit/test_utils.py

```
@@ -346,50 +347,62 @@ def test_incomplete_sequencing_runs_are_skipped(self):
 
         rmtree(sequencer_output_dir)
 
-    def test_runs_over_max_age_skipped(self):
+    def test_checking_run_age_against_max_age_is_correct(self):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_checking_run_age_against_max_age_is_correct`

Add return type annotation: `None`

(ANN201)